.babelrc.json

```diff
@@ -13,8 +13,7 @@
       ],
       "env": {
         "cjs": {
-          "presets": [["@babel/preset-env", { "modules": "commonjs" }]],
-          "plugins": ["./resources/inline-invariant"]
```

Comment:
This plugin doesn't exist. Since the babel config wasn't used by the build script, it wasn't erroring out.